implementations/rust/ockam/ockam_node/src/router/utils.rs

```diff
@@ -34,6 +34,7 @@ pub(super) async fn resolve(
     match router.map.internal.get(&primary_address) {
         Some(record) if record.check() => {
             trace!("{} OK", base);
+            record.increment_msg_count();
```

Comment:
I guess technically it's correct for now, but I'm surprised that 1 address resolution == 1 message sent